README.md

```diff
@@ -20,13 +20,17 @@ This app demonstrates all of the WinUI 3 controls and styles available to make a
 </a>
 </p>
 
-## ⭐ Features:
+## ⭐ Features
+
+- **WinUI controls samples:** each control page shows the markup and codebehind used to create each example.
+
+- **Use the Microsoft.UI.Xaml (WinUI) library:** the app includes the latest WinUI NuGet package and shows how to use the [WinUI](https://docs.microsoft.com/windows/apps/winui/) controls like NavigationView, SwipeControl, and more.
 
-- **WinUI Controls Samples:** Each control page shows the markup and codebehind used to create each example.
-- **Use the Microsoft.UI.Xaml (WinUI) Library:** The app includes the latest WinUI NuGet package and shows how to use the [WinUI](https://docs.microsoft.com/windows/apps/winui/) controls like NavigationView, SwipeControl, and more.
 - **Basic Layout:** This sample will show all of the possible layout options for your app and allow you to interact with the panels to show how to achieve any layout you are looking for.

```

Comment:
```suggestion
```

We don't have such sample (anymore) as it's part of many samples. Feels weird to call this out explicitly?